### ISSUE TITLE
Bad handling of non-ASCII characters in social networks names

### ISSUE THREAD

**Issue by bortzmeyer** [andstatus/andstatus#452]
A social network name with accented characters is accepted, but then the non-ASCII characters is displayed as a dot.

**Comment by yvolk** [comment]
Yes, this is by design. Code explicitly does this substitution, see the line 476 here: https://github.com/andstatus/andstatus/blob/master/app/src/main/java/org/andstatus/app/origin/Origin.java
The name is used as a part of Android Account name (as a key...), maybe this is why I did this name cleaning.
